pages/linux/at.md

```diff
@@ -0,0 +1,15 @@
+# at
+
+> Executes commands at a specified time.
+
+- Open `at` prompt to execute commands at specific time. When you're done, press Ctrl+D to save the schedule:
+
+`at {{hh:mm:ss}}`
+
+- Execute commands and email result:
+
+`at {{hh:mm:ss}} -m`
+
+- Execute script:
+
+`at {{hh:mm:ss}} -f {{/path/to/file}}`
```

Comment:
please remove the leading / in the path, since it doesn't have to be an absolute path (this is the convention used in other pages)